### ISSUE TITLE
test Remount Secure in Qubes

### ISSUE THREAD

**Issue by adrelanos** [Kicksecure/security-misc#137]

**Comment by adrelanos** [comment]
This is blocked by my inability to make dracut work inside Qubes App Qubes. Reported upstream:

* https://github.com/QubesOS/qubes-issues/issues/8649

**Comment by adrelanos** [comment]
The blocker was resolved.

**Comment by adrelanos** [comment]
new plan:
https://github.com/Kicksecure/security-misc/issues/157#issuecomment-1840229516
